packages/expo-modules-core/ios/Fabric/ExpoFabricView.swift

```diff
@@ -118,6 +118,17 @@ open class ExpoFabricView: ExpoFabricViewObjC, AnyExpoView {
 
   // MARK: - Statics
 
+  /**
+   Called by React Native to check if the view supports recycling.
+   */
+  @objc
+  public static func shouldBeRecycled() -> Bool {
```

Comment:
since it's `static`, so people may not have an easy way to override this, do you have any better solution to override `shouldBeRecycled`?

Reply:
Ah, you're right. Looks like even with `open class func` it's not possible to override it 🤔

Reply:
So I decided to just make it disabled without an option to enable it. Let's work on it later.